### ISSUE TITLE
[firewareos] prompt regexp invalid

### ISSUE THREAD

**Issue by mikenowak** [ytti/oxidized#813]
Introduced by #810.

As a result, current regexp matches some of the tags in the xml - i.e. `<wgsync>`, which then cause only a part of the config to be stored, and the rest is discarded.

Looking at ssh debug file, I confirm that complete config is being returned to oxidized.

**Comment by ytti** [comment]
Should I expect that you're working to make the regexp more specific, to avoid spurious matches?

**Comment by mikenowak** [comment]
Yes, I am looking at this now:

```
$ ruby -pe 'next unless $_ =~ /^\[?\w*\]?\w*<?\w*>?(#|>)\s*$/' < my-device-config-file
<profile>
```

This matches the very beginning of the config file returned by `export config to console`

**Comment by wk** [comment]
Closing this issue as the fix appears to have been merged in #815
